frontends/mit-open/src/routes.tsx

```diff
@@ -37,6 +38,10 @@ const routes: RouteObject[] = [
         path: urls.LEARNINGPATH_VIEW,
         element: <LearningPathDetailsPage />,
       },
+      {
+        path: urls.USERLIST_LISTING,
+        element: <UserListListingPage />,
+      },
```

Comment:
**Suggestion:** I think it makes sense to only show the userlist listing page to authenticated users. 
```tsx
      {
        element: <RestrictedRoute requires={Permissions.Authenticated} />,
        children: [
          {
            path: urls.USERLIST_LISTING,
            element: <UserListListingPage />,
          },
          {
            path: urls.DASHBOARD,                        @CAREY and remove the existing dashboard route
            element: <DashboardPage />,
          },
        ],
      },
```

The details page could be shown to everyone. Userlists can be "private" (only for that user) or "unlisted" (public, but not searchable).

Reply:
I thought this was handled in the same way the permissions for the Learning Paths menu item was in `frontends/mit-open/src/page-components/Header/UserMenu.tsx`?

```
...
    {
      label: "User Lists",
      key: "userlists",
      allow: hasPermission(Permissions.Authenticated),
      href: urls.USERLIST_LISTING,
    },
...
```

Reply:
Oh, I see, that file just prevents it from showing up in the menu and `routes.tsx` prevents it from being routed to entirely

Reply:
>  Should we do this with the Learning Path listing page as well?

I'm not sure. At some point that will be a product question.

Right now, an anonymous user visiting http://localhost:8063/learningpaths/ will see a list of all **public** learning paths (b/c that's what the API returns for anonymous users).